arch/arm64/configs/aosp_nile_discovery_defconfig

```diff
@@ -1,5 +1,5 @@
 # CONFIG_USELIB is not set
-CONFIG_AUDIT=y
+CONFIG_AUDIT=n
```

Comment:
As per https://github.com/mer-hybris/mer-kernel-check/blob/master/mer_verify_kernel_config#L129, please leave AUDIT enabled since you're controlling SELinux via the BOOTPARAM flag below.

If you're in doubt, just check how pioneer has been done: https://github.com/mer-hybris/android_kernel_sony_msm/blob/hybris-sony-aosp-8.1.0_r35_20180714/arch/arm64/configs/aosp_nile_pioneer_defconfig#L2